selfservice/flow/state.go

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package flow
+
+// Flow State
+//
+// The state represents the state of the verification flow.
+//
+// - choose_method: ask the user to choose a method (e.g. recover account via email)
+// - sent_email: the email has been sent to the user
+// - passed_challenge: the request was successful and the recovery challenge was passed.
+// - show_form: a form is shown to the user to perform the flow
+// - success: the flow has been completed successfully
+//
+// swagger:enum SelfServiceFlowState
```

Comment:
this needs to be of type "enum" and the ones aliasing this needs to be of type "model" otherwise go-swagger crashes when generating the sdk :shrug: 
we then manually patch the other flows aliasing this type here as type enum.